production/scrapers/sacramento_county_jail.R

```diff
@@ -19,32 +19,34 @@ sacramento_county_jail_extract <- function(x, exp_date = Sys.Date()){
     
     check_names(x, c(
         "As of Date", 
-        "Cumulative Confirmed Cases - Both Jails", 
-        "Cumulative Confirmed Cases - Intake/Quarantine Period - Both Jails", 
-        "Active in Custody - Main Jail", 
-        "Active in Custody - RCCC", 
-        "New Active Cases - Main Jail", 
-        "New Active Cases - RCCC", 
-        "Released while Active", 
-        "Resolved", 
-        "Deaths", 
-        "Cumulative Tested - Both Jails", 
-        "New Tests Administered - Both Jails", 
-        "Total Population - Both Jails", 
-        "Population Change - Both Jails", 
-        "% Population Newly Tested - Both Jails", 
+        "Facility Name", 
+        "County", 
+        "Confirmed Cases (Incarcerated population, cumulative)", 
+        "Cases Confirmed at Intake or Quarantine Period (Incarcerated population, cumulative)", 
+        "Active Cases (Incarcerated population, current, ONLY Main Jail)", 
+        "Active Cases (Incarcerated population, current, ONLY RCCC)", 
+        "Active Cases (Incarcerated population, current)", 
+        "Active Cases (Incarcerated population, 7-day diff, ONLY Main Jail)", 
+        "Active Cases (Incarcerated population, 7-day diff, ONLY RCCC)", 
+        "Active Cases (Incarcerated population, 7-day diff)", 
+        "Cases Released while Active (Incarcerated population)", 
+        "Deaths (Incarcerated population, cumulative)", 
+        "Tests (Incarcerated population, cumulative)", 
+        "New Tests (Incarcerated population, 7-day diff)", 
+        "Population (Incarcerated population, current)", 
+        "Population (Incarcerated population, 7-day diff)", 
+        "Percent of Population Tested (Incarcerated population, 7-day period)", 
         "Notes", 
```

Comment:
It looks like some of the column names have changed but I think we should leave it given that they might change again :/

```
Warning messages:
1: In basic_check(true_names, expected_names) :
  Extracted column 6 does not match expected name. Expected: Active Cases Incarcerated population current ONLY Main Jail Received: Active Cases Incarcerated population current Main Jail
2: In basic_check(true_names, expected_names) :
  Extracted column 7 does not match expected name. Expected: Active Cases Incarcerated population current ONLY RCCC Received: Active Cases Incarcerated population current RCCC
3: In basic_check(true_names, expected_names) :
  Extracted column 9 does not match expected name. Expected: Active Cases Incarcerated population 7-day diff ONLY Main Jail Received: Active Cases Incarcerated population 7-day diff Main Jail
4: In basic_check(true_names, expected_names) :
  Extracted column 10 does not match expected name. Expected: Active Cases Incarcerated population 7-day diff ONLY RCCC Received: Active Cases Incarcerated population 7-day diff RCCC
```

Reply:
> Made a minor change in the sf jail col names which im guessing was because they once again edited names of the columns.
> 
> Not directly related to this PR but it looks like we might be missing data for extractions when they fill in a date and name but dont put in values. Do we want to account for this and instead grab the last date where we have a conformed case count?
> 
> ![2021-04-15-103155_1786x309_scrot](https://user-images.githubusercontent.com/5198764/114887263-47e9d100-9dd6-11eb-884e-67d59469b4fd.png)

Yup! Either that or forward-fill all of the columns we scrape within a window since it seems like they collect different variables on different dates (e.g. #196).